packages/openapi-v3-types/LICENSE

```diff
@@ -0,0 +1,25 @@
+Copyright (c) IBM Corp. 2017,2018. All Rights Reserved.
+Node module: @loopback/openapi-spec
```

Comment:
Please fix the module name.

Ideally, I am recommending to use `slt license` and `slt copyright` to fix all these things automatically for you.

```
$ SLT_OWNER="IBM Corp." slt copyright
$ SLT_OWNER="IBM Corp." slt license
```